sdk/tables/data-tables/tests.yml

```diff
@@ -1,5 +1,8 @@
 trigger: none
 
+variables:
+    jobMatrixFilter: ^((?!8x_node).)*$
```

Comment:
@joheredi this variable is generally intended for runtime/UI only. It would be better to add another filter to the `MatrixFilters` parameter already in this file:

```
parameters:
...
      MatrixFilters:
        - DependencyVersion=^$
        - NodeTestVersion=^(?!8.x).*
```